pkg/orchestrator/scanestimationwatcher/watcher.go

```diff
@@ -512,6 +515,23 @@ func (w *Watcher) reconcileInProgress(ctx context.Context, scanEstimation *model
 	return nil
 }
 
+func updateTotalScanTimeWithParallelScans(scanEstimation *models.ScanEstimation) {
```

Comment:
Should this function actually return an `error` if the `scanEstimation` is `nil` or one of its fields like `ScanTemplate` or `Summary`? And let the caller to decide how to handle the error?

Reply:
I can add an error return here in that case.